src/utils/fetchOpenSaucedApiData.ts

```diff
@@ -177,6 +178,14 @@ export const updateInsight = async (userToken: string, repoId: string, checked:
 
     return response.status === 200;
 };
+export const getHighlights = async () => {
+    const response = await fetch(
+        `${OPEN_SAUCED_HIGHLIGHTS_ENDPOINT}`,
```

Comment:
Isn't there a limit to how many highlights this will return (idr)?

We could either paginate this or simply show a hint like:
1/10
on the home page

Reply:
The limit is 10 and we don't need to paginate for more. The hope is the latest 10 is enough and the user and view the rest within the app.